CSharp/AccountManagement/Program.cs

```diff
@@ -249,30 +263,31 @@ private static string PromptUserForAccountName()
         /// <summary>
         /// Performs various Batch account operations using the Batch Management library.
         /// </summary>
-        /// <param name="creds">The <see cref="Microsoft.Azure.TokenCloudCredentials"/> containing information about the user's
-        /// Azure account and subscription.</param>
+        /// <param name="accessToken">The access token to use for authentication.</param>
+        /// <param name="subscriptionId">The subscription id to use for creating Batch management client</param>
         /// <param name="location">The location where the Batch account will be created.</param>
         /// <returns>A <see cref="System.Threading.Tasks.Task"/> object that represents the asynchronous operation.</returns>
-        private static async Task PerformBatchAccountOperationsAsync(TokenCloudCredentials creds, string location)
+        private static async Task PerformBatchAccountOperationsAsync(string accessToken, string subscriptionId, string location)

```

Comment:
On Line 274, the subscription id is set for the batch management client. Subscription id is a required parameter in the swagger spec that needs to be set first before calling any batch account oeprations